.travis.yml

```diff
@@ -15,6 +15,7 @@ jobs:
         - psql -c "CREATE USER chroma;" -U postgres
         - psql -c "ALTER USER chroma CREATEDB;" -U postgres
         - psql -c "CREATE DATABASE chroma OWNER chroma;" -U postgres
+        - psql -c 'CREATE EXTENSION IF NOT EXISTS btree_gist;' -U postgres -d template1
```

Comment:
Why is this not created for the chroma db?

Reply:
Because testing harness creates a new DB for testing, and we don't control it, so we enable extension for template DB used to create the testing DB.